server_environment/serv_config.py

```diff
@@ -29,29 +30,37 @@
 
 from .system_info import get_server_environment
 
-from openerp.addons import server_environment_files
-_dir = os.path.dirname(server_environment_files.__file__)
+_logger = logging.getLogger(__name__)
 
 # Same dict as RawConfigParser._boolean_states
 _boolean_states = {'1': True, 'yes': True, 'true': True, 'on': True,
                    '0': False, 'no': False, 'false': False, 'off': False}
 
-if not system_base_config.get('running_env', False):
```

Comment:
Why not move this logic (L39:L54) into `_load_config()` and then wrap the `serv_config = _load_config()` in a `try/except` for EnvironmentError?